cegs_portal/search/views/v1/non_targeting_reos.py

```diff
@@ -71,17 +87,21 @@ def get_sig_only(value):
         return options
 
     def get(self, request, options, data, feature_id):
-        reo_page, feature = data
-        response_values = {"non_targeting_reos": reo_page, "feature": feature}
-
-        if request.headers.get("HX-Target"):
-            return render(
-                request,
-                "search/v1/partials/_non_targeting_reo.html",
-                response_values,
-            )
-
-        return super().get(request, options, response_values)
+        if request.headers.get('Accept') == TSV_MIME:
```

Comment:
This shouldn't be necessary, that's what modifying the TemplateJsonView takes care of.